pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -63,41 +71,83 @@ func (pp *predicatesPlugin) Name() string {
 	return PluginName
 }
 
+type baseResource struct {
+	CPU    int
+	Memory int
+}
```

Comment:
I think it will be better to support float

Reply:
ok, I changed it to float64.